packages/docs/src/routes/docs/components/styles/index.mdx

```diff
@@ -53,7 +53,7 @@ Component styles allow Qwik to lazy load the style information for the component
 
 ```tsx
 import {useStyles$} from '@builder.io/qwik';
-import styles from './code-block.css?inline';
```

Comment:
i saw you reverted this! it was correct

Reply:
omg! nice catch! wasn't on purpose 🙈  reverted 👍